content/english/python-earsketch/activity-1-set-tempo-add-sounds.md

```diff
@@ -7,60 +7,61 @@ weight: 5
 draft: false
 ---
 
-<iframe width="100%" height="600px" src="https://www.youtube.com/embed/3OpCc7sKcpM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<p style="text-align: center;"><iframe width="60%" height="600px" src="https://www.youtube.com/embed/3OpCc7sKcpM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
 
 Let's take a closer look at the pre-existing code and understand what it means. ![annotated screenshot cannot be displayed](../img/annotated-screenshot-overview.png) We will be placing our new code between the `setTempo()` and `finish()` functions. The `setTempo()` function allows you to set the overall project tempo.
 
 {{% notice info %}}
-**Quick Fact**: Tempo is the speed at which a piece of music is played. Changing a project's tempo allows for different styles of music to be created. Try changing the tempo of your `setTempo` function and see what happens! Please make sure it is a number
-between 45-220. 
+
+**Quick Fact**: Tempo is the speed at which a piece of music is played. Changing a project's tempo allows for different styles of music to be created. Try changing the tempo of your `setTempo` function and see what happens! Please make sure it is a number between 45-220.
+
 ![](../img/img-tempo1.png)
+
 {{% /notice %}}
 
+## Activity 1
 
 1. Now that our tempo is set, it is time to add sounds. To do so, we must utilize the `fitMedia()` function.
+
```

Comment:
For consistency, let's remove the new lines added between the 1., 2. etc.

Reply:
updated